src/ts/plugins/server/GitlabHostingConnector.ts

```diff
@@ -180,9 +200,10 @@ export default class GitlabHostingConnector extends GitlabConnector implements H
     } catch (error) {
       console.error('Error during creating new tag:', error.message)
       jobError(job.jobId, `Failed to create new tag: ${error.message}`)
-      return false
+      return 'Failed'
```

Comment:
I suggest to return null in case there is no tag
So the function will have a return type of string | null

Reply:
and tested at the beginning with `if (!successTag)` instead of  `if (successTag === 'Failed')` ?